assets/src/edit-story/components/form/numeric.js

```diff
@@ -79,6 +79,49 @@ const Suffix = styled.span`
     `}
 `;
 
+const DisplayMode = {
+  Empty: 0,
+  Value: 1,
```

Comment:
Nit: perhaps `DISPLAY_MODE`, `EMPTY`, `VALUE` for consistency with constants used elsewhere.